cpp/include/cudf/io/detail/parquet.hpp

```diff
@@ -74,7 +75,7 @@ class reader {
    * @return The set of columns along with table metadata
    */
   table_with_metadata read(parquet_reader_options const& options,
-                           rmm::cuda_stream_view stream = rmm::cuda_stream_default);
+                           rmm::cuda_stream_view stream = cudf::default_stream_value);
```

Comment:
`rmm::cuda_stream_default` already has the meaning you want when per-thread default stream is enabled.